src/lumigo_opentelemetry/__init__.py

```diff
@@ -94,9 +98,20 @@ def init():
     from lumigo_opentelemetry.instrumentations import instrumentations  # noqa
     from lumigo_opentelemetry.instrumentations.instrumentations import framework
     from lumigo_opentelemetry.resources.detectors import get_resource
+    from lumigo_opentelemetry.libs.json_utils import DEFAULT_MAX_ENTRY_SIZE
```

Comment:
It does not feel right for the `DEFAULT_MAX_ENTRY_SIZE` to be in a util file about json (at least judging by its name). We need to reorg this code I think.